data/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+go_ontology: &id1
```

Comment:
Add a comment in here with `#` that states the purpose of this file (something like "Configure default query field values for each namespace")